<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKChartsXYSerialWidget</class>
 <widget class="QWidget" name="SAKChartsXYSerialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形图</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QWidget" name="chartViewerWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="closePushButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKChartsXYSerialWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
